AztecTests/Extensions/NSAttributedStringParagraphRangeTests.swift

```diff
@@ -0,0 +1,40 @@
+import XCTest
+@testable import Aztec
+
+class NSAttributedStringParagraphRangeTests: XCTestCase {
+    
+    func testParagraphRangesWithoutSeparator() {
```

Comment:
WDYT about appending `SpanningZeroPointRange` (or something similar) to each these test names? I'd imagine in the future we may have other tests that ensure everything works fine for strings with paragraphs spanning ranges greater than zero?

Reply:
Added documentation to each test, and a more precise name to describe what's being tested.